recipe/recipe.yaml

```diff
@@ -63,21 +73,39 @@ build:
       then: |
         echo hello
         set LLAMA_ARGS=-DLLAMA_BUILD_TESTS=OFF
+
         {% macro llama_args(value) -%}
         set LLAMA_ARGS=%LLAMA_ARGS% -DLLAMA_${{ value }}
         {%- endmacro %}
+
         {% macro cmake_args(value) -%}
         set LLAMA_ARGS=%LLAMA_ARGS% -D${{ value }}
         {%- endmacro %}
 
+        {% macro ggml_args(value) -%}
+        set LLAMA_ARGS=%LLAMA_ARGS% -DGGML_${{ value }}
+        {%- endmacro %}
+
         ${{ cmake_args("BUILD_SHARED_LIBS=ON") }}
-        ${{ llama_args("NATIVE=OFF") }}
+        ${{ llama_args("CURL=OFF") }}
+
+        ${{ ggml_args("NATIVE=OFF") }}
+
+        {%- if cuda_compiler_version != "None" %}
+        ${{ ggml_args("CUDA=ON") }}
+
+        :: NOTE: is it necessary to set `CMAKE_CUDA_ARCHITECTURES=all`
+        :: on Windows, or is it a nice-to-have?
+        :: ${{ cmake_args("CMAKE_CUDA_ARCHITECTURES=all") }}
```

Comment:
It seems that [this was used before](https://github.com/conda-forge/llama.cpp-feedstock/pull/32/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR45-R46). Could it be the reason for the timeout of Windows?